04-rstudio-tour.Rmd

```diff
@@ -1,38 +1,277 @@
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Whe pane on the lower right is where we see our files, plots, and where we get help."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g20ed7630a13_1_109")
+```
+
+
+While there are other tabs, don't worry about those for now. We will go deeper into RStudio as we continue. 
+
+
+Let's try an example to get started.
+
+As an example, we could type in the code `head(iris)` and press enter to see the code execute and preview.
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Running some code in RStudio"}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g20ed7630a13_1_142")
+```
+
+Now let's try another example where instead of just printing some data to the screen we assign a data object that will show up in the environment using the `<-` notation. This is useful in a situation if we want to modify the iris data somehow but want to keep the original version.
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "If you create an object in RStudio, it will show up in the environment pane."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g1fa1583c827_0_4")
+```
+
+### The Hidden Pane
+
+There is a hidden fourth pane. This is only accessible when we start to make a script or a report with our code.
+
+To open this let's make what is called an R Markdown file by go to `File` --> `New File` -->`R Markdown` in the upper menu of RStudio.
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Creating an R Markdown file in R Studio by selecting File, New File, R Markdown."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g20ed7630a13_1_127")
+```
+
+Creating an R Markdown file starts with a pop-up and you can simply click the OK button.
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Creating an R Markdown file starts with a pop-up and you can simply click the OK button."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g20ed7630a13_1_169")
+```
+
+The new pane will open on the upper left.
+
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "The new pane will open on the upper left."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g20ed7630a13_1_176")
+```
+
+This pane is where we can write code that we keep in files like scripts or reports (in files like R Markdowns).
+
+Thus the lower left pane is where we can test out code, but the top pane is where we can write code that we wish to save. 
```

Comment:
```suggestion
Thus the lower left pane is where we can test out code, but the top pane is where we can write code that we wish to save. 
```

I think we should try to move folks away from testing things in the console per se. Instead I thinking trying to have them highlight code in their script/notebook and run from there is better and more reproducible. People new to R have a tendency of writing everything in the console and underestimating how many steps they did outside their notebook that are not recorded and therefore not reproducible.

Reply:
I agree - but thought that the next chapter would really be focused on this, but I will add a statement here about that